pvlib/iotools/solcast.py

```diff
@@ -111,7 +114,7 @@ def get_solcast_tmy(
         map_variables=map_variables
     )
 
-    return data, {}
+    return data, {"latitude": latitude, "longitude": longitude}

```

Comment:
Do/can we also get site elevation back, and should we return that as metadata as well?